cmd/status/cmdstatus.go

```diff
@@ -109,6 +111,17 @@ func (r *Runner) runE(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
+	printData := printer.PrintData{}
+	// initialize maps in printData
+	printData.InvNameMap = make(map[object.ObjMetadata]string)
```

Comment:
I think it is more idomatic go to do the initialization while you are creating printData. So:
```golang
printData := printer.PrintData{
  InvNameMap: make(map[object.ObjMetadata]string),
  StatusSet: make(map[string]bool),
}
```